addons/mass_mailing/wizard/mailing_mailing_test.py

```diff
@@ -32,6 +36,7 @@ def send_mail_test(self):
                 invalid_candidates.append(candidate)
 
         mailing = self.mass_mailing_id
+        mailing.last_test_recipients_email = self.email_to
```

Comment:
To check if with the related, we still need this line (normally no, but since we are in a wizard I have a doubt about the propagation)